spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -106,8 +110,8 @@ FungibleTokenPacketDataV2 {
   ],
   sender: cosmosexampleaddr1,
   receiver: cosmosexampleaddr2,
-  memo: "exampleMemo",
-  forwardingPath: [{"transfer", "channel-7", ""}, {"transfer", "channel-13", "swap: {...}"}],
+  memo: "",
+  forwardingPath: {[{"transfer", "channel-7"}, {"transfer", "channel-13"}], , "swap: {...}"}, // provide hops in order and the memo intended for final hop
```

Comment:
Is the double `,` close to swap in line 114 intentional?  Should it be `forwardingPath: {[{"transfer", "channel-7"}, {"transfer", "channel-13"}], "swap: {...}"}, // provide hops in order and the memo intended for final hop` ?